src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -124,6 +124,11 @@ public IReadOnlyList<string> GetEffectiveFallbackPackageFolders(ISettings settin
             return SettingsUtility.GetFallbackPackageFolders(settings);
         }
 
+        public bool GetForceUpdateLastAccessTime(ISettings settings)
```

Comment:
I don't think we need this method. It doesn't carry a lot of value as a method on the RestoreArgs class.

Reply:
Removed.